src/stylesheets/button.scss

```diff
@@ -83,7 +83,7 @@ button.btn, a.btn {
   }
   &.centered {
     margin: auto;
-    font-size: 1.2vw;
+    font-size: 1.8vw;
```

Comment:
For some reason this looked very small!
![Screen Shot 2019-09-05 at 11 00 39 AM](https://user-images.githubusercontent.com/1280564/64372484-61b71580-cff0-11e9-8122-f660e00e77a1.png)